docs/refactor-migrate-graphql-to-rest.md

```
@@ -0,0 +1,51 @@
+# Migrate GraphQL/Ferry to REST
+
+- **PR:** not yet opened
+- **Branch:** refactor/migrate-graphql-to-rest → main
+- **Date:** 2026-07-08 13:13
+- **Author:** Enzo Lizama
+
+## Summary
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)